src/types.ts

```diff
@@ -1,20 +1,10 @@
+import { addressSchema, keccak256HashSchema } from '@api3/commons';
 import { z } from 'zod';
 
-import { type EvmAddress, evmAddressSchema, type EvmId, evmIdSchema } from './config/schema';
-
-export type AirnodeAddress = EvmAddress;
-export type TemplateId = EvmId;
-export type PrivateKey = string;
-export type BeaconId = EvmId;
-export type ChainId = string;
-export type ProviderName = string;
-export type SignedApiUrl = string;
-export type DapiNameOrDataFeedId = string;
-
 // Taken from https://github.com/api3dao/signed-api/blob/main/packages/api/src/schema.ts
 export const signedDataSchema = z.object({
-  airnode: evmAddressSchema,
-  templateId: evmIdSchema,
+  airnode: addressSchema,
+  templateId: keccak256HashSchema,
   timestamp: z.string(),
   encodedValue: z.string(),
```

Comment:
[hexSchema](https://github.com/api3dao/commons/blob/fb07e2c083cf63bc35d500aca37eb3fb7ee411c8/src/blockchain-utilities/schema.ts#L5) for `encodedValue` and `signature`?